src/app/components/search-bar/search-bar.component.ts

```diff
@@ -24,7 +25,7 @@ export class SearchBarComponent implements OnInit{
   }
 
   private initFields(): void {
-    this.query = new FormControl('', [Validators.required]);
+    this.query = new FormControl('');
```

Comment:
Why did you remove the required validation?

Reply:
it just felt redundant since there is no submit for the query, the filter invokes automatically on input change